features/networking/multus-ipv6.feature

```diff
@@ -50,3 +50,70 @@ Feature: Multus-CNI ipv6 related scenarios
       | curl | -g | -6 | --connect-timeout | 5 | [<%= cb.pod1_multus_ipv6 %>]:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-38521
+  @admin
+  Scenario: IPv6 testing for OCP-31999: Whereabouts should exclude IPv6 ranges
+  # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1913062
+  # Bug https://bugzilla.redhat.com/show_bug.cgi?id=1917984
+  # Make sure that the multus is enabled
+    Given the master version >= "4.6"
+    And the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+    And evaluation of `node.name` is stored in the :target_node clipboard
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    Given I obtain test data file "networking/multus-cni/NetworkAttachmentDefinitions/whereabouts-excludeIP.yaml"
+    When I run oc create as admin over "whereabouts-excludeIP.yaml" replacing paths:
+      | ["metadata"]["name"]      | whereabouts-excludeipv6                                                                                                                                                                                                  |
+      | ["metadata"]["namespace"] | <%= project.name %>                                                                                                                                                                                                      |
+      | ["spec"]["config"]        | '{ "cniVersion": "0.3.0", "name": "whereabouts", "type": "macvlan", "mode": "bridge", "ipam": { "type": "whereabouts", "range": "fd00:dead:beef:1::1-fd00:dead:beef:1::4/64", "exclude": ["fd00:dead:beef:1::2/128"] } }'|
```

Comment:
Is there a line length limit at which we can break this line to new line? Or it has to be on this line? Curious, as in Python we enforced line break after 80 characters.

Reply:
@kedark3  Good question, I am not really sure about the characters limitation for our cucushif testing, just follow the existing scripts format. Other forks may know it.

Reply:
Thanks @weliang1 I would definitely like to see if we can break this line. I am new to this repo so won't be able to tell if we can do that :man_shrugging:

Reply:
@kedark3 @weliang1 this has the coding guideline https://github.com/openshift/verification-tests/blob/master/doc/coding_guide.adoc. With tables, we align the `|` and historically we don't break the line even though they go over 80 characters sometimes.

Reply:
Thanks @pruan-rht